src/adapters/hasura/altStudent.adapter.ts

```diff
@@ -1041,4 +1050,138 @@ export class ALTStudentService {
       data: responseData,
     });
   }
+  public async updateStudent(userId, request, body) {
+    const decoded: any = jwt_decode(request.headers.authorization);
+    const altUserRoles =
+      decoded["https://hasura.io/jwt/claims"]["x-hasura-allowed-roles"];
+
+    const studentFields = [
+      "groups",
+      "religion",
+      "caste",
+      "annualIncome",
+      "motherEducation",
+      "fatherEducation",
+      "motherOccupation",
+      "fatherOccupation",
+      "noOfSiblings",
+      "schoolUdise",
+      "board",
+      "state",
+      "block",
+      "district",
+    ];
+
+    const userFields = ["name", "email", "gender", "dateOfBirth", "mobile"];
+    let userUpdate = "";
+    let studentUpdate = "";
+    let userUpdateFields = "";
+    let studentUpdateFields = "";
+
+    Object.keys(body).forEach((field) => {
```

Comment:
@YoginiTayade Add comments in each section for better code redability

Reply:
resolved